app/management/commands/update_ungm.py

```diff
@@ -119,14 +122,17 @@ def update_ungm_tenders(parsed_tenders):
                                 setattr(tender_doc, k, v)
 
                         tender_doc.save()
+                        self.download_document(tender_doc)
 
                     except TenderDocument.DoesNotExist:
```

Comment:
You can use
finally: 
   self.download_document(new_doc)

and it executes for both except and try